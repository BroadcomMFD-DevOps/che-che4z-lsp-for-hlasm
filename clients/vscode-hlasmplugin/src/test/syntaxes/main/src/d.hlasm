LABEL INSTR      O,                                                    -
               O                           A(GDT)                      -
               O,                                                      -
                                                                       -
               OP1 REMARKS                                             -
               OP1 REMARKS                                             -
               REMARKS 2                                                12345678
.L20     MNOTE ,,'llll'

.L20     MNOTE 4,                                                      X12345678
               '  Length of block does not exceed 256 (&LBL DSECT), &&CX12345678
            LEANUP option can be optimized to YES',OP2OP2OP2OP2OP2O,   -12345678
               REMARK REMARK

.L20     MNOTE 4,                                                      X12345678
               '  Length of block does not exceed 256 (&LBL DSECT), &&CX12345678
        Length of block does not exceed 256 (&LBL DSECT), &&C          X12345678
               LEANUP option can be optimized to YES',OP2OP2OP2OP2OP2O,-12345678
               REMARK REMARK