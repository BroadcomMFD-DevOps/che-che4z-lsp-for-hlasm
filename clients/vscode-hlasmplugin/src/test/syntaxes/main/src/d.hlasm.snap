>LABEL INSTR      O,                                                    -
#^^^^^ source.hlasm support.class.hlasm
#     ^^^^^^ source.hlasm entity.name.function.hlasm
#           ^^^^^^ source.hlasm
#                 ^ source.hlasm punctuation.definition.tag.hlasm
#                  ^ source.hlasm
#                   ^ source.hlasm
#                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm
#                                                                       ^ source.hlasm keyword.other.hlasm
>               O                           A(GDT)                      -
#^^^^^^^^^^^^^^^ source.hlasm
#               ^ source.hlasm punctuation.definition.tag.hlasm
#                ^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm
#                                           ^^^^^^ source.hlasm comment.hlasm
#                                                 ^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm keyword.other.hlasm
>               O,                                                      -
#^^^^^^^^^^^^^^^ source.hlasm
#               ^^ source.hlasm comment.hlasm
#                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm keyword.other.hlasm
>                                                                       -
#^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm keyword.other.hlasm
>               OP1 REMARKS                                             -
#^^^^^^^^^^^^^^^ source.hlasm
#               ^^^ source.hlasm comment.hlasm
#                  ^ source.hlasm
#                   ^^^^^^^ source.hlasm comment.hlasm
#                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm keyword.other.hlasm
>               OP1 REMARKS                                             -
#^^^^^^^^^^^^^^^ source.hlasm
#               ^^^ source.hlasm comment.hlasm
#                  ^ source.hlasm
#                   ^^^^^^^ source.hlasm comment.hlasm
#                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm keyword.other.hlasm
>               REMARKS 2                                                12345678
#^^^^^^^^^^^^^^^ source.hlasm
#               ^^^^^^^ source.hlasm comment.hlasm
#                      ^ source.hlasm
#                       ^ source.hlasm comment.hlasm
#                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm invalid.ignored
>.L20     MNOTE ,,'llll'
#^^^^ source.hlasm keyword.control.hlasm
#    ^^^^^^^^^^ source.hlasm entity.name.function.hlasm
#              ^ source.hlasm
#               ^^ source.hlasm
#                 ^ source.hlasm string.hlasm
#                  ^^^^^ source.hlasm string.hlasm
>
>.L20     MNOTE 4,                                                      X12345678
#^^^^ source.hlasm keyword.control.hlasm
#    ^^^^^^^^^^ source.hlasm entity.name.function.hlasm
#              ^ source.hlasm
#               ^ source.hlasm punctuation.definition.tag.hlasm constant.numeric.hlasm
#                ^ source.hlasm
#                 ^ source.hlasm
#                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm
#                                                                       ^ source.hlasm keyword.other.hlasm
#                                                                        ^^^^^^^^ source.hlasm invalid.ignored
>               '  Length of block does not exceed 256 (&LBL DSECT), &&CX12345678
#^^^^^^^^^^^^^^^ source.hlasm
#               ^ source.hlasm string.hlasm
#                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm string.hlasm
#                                                       ^^^^ source.hlasm variable.other.readwrite.hlasm
#                                                           ^^^^^^^^^ source.hlasm string.hlasm
#                                                                    ^^ source.hlasm string.hlasm
#                                                                      ^ source.hlasm string.hlasm
#                                                                       ^ source.hlasm keyword.other.hlasm
#                                                                        ^^^^^^^^ source.hlasm invalid.ignored
>            LEANUP option can be optimized to YES',OP2OP2OP2OP2OP2O,   -12345678
#^^^^^^^^^^^^^^^ source.hlasm invalid.illegal
#               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm string.hlasm
#                                                  ^ source.hlasm
#                                                   ^^^^^^^^^^^^^^^^ source.hlasm punctuation.definition.tag.hlasm
#                                                                   ^ source.hlasm
#                                                                    ^ source.hlasm
#                                                                     ^^ source.hlasm
#                                                                       ^ source.hlasm keyword.other.hlasm
#                                                                        ^^^^^^^^ source.hlasm invalid.ignored
>               REMARK REMARK
#^^^^^^^^^^^^^^^ source.hlasm
#               ^^^^^^ source.hlasm punctuation.definition.tag.hlasm
#                     ^ source.hlasm
#                      ^^^^^^ source.hlasm comment.hlasm
>
>.L20     MNOTE 4,                                                      X12345678
#^^^^ source.hlasm keyword.control.hlasm
#    ^^^^^^^^^^ source.hlasm entity.name.function.hlasm
#              ^ source.hlasm
#               ^ source.hlasm punctuation.definition.tag.hlasm constant.numeric.hlasm
#                ^ source.hlasm
#                 ^ source.hlasm
#                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm
#                                                                       ^ source.hlasm keyword.other.hlasm
#                                                                        ^^^^^^^^ source.hlasm invalid.ignored
>               '  Length of block does not exceed 256 (&LBL DSECT), &&CX12345678
#^^^^^^^^^^^^^^^ source.hlasm
#               ^ source.hlasm string.hlasm
#                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm string.hlasm
#                                                       ^^^^ source.hlasm variable.other.readwrite.hlasm
#                                                           ^^^^^^^^^ source.hlasm string.hlasm
#                                                                    ^^ source.hlasm string.hlasm
#                                                                      ^ source.hlasm string.hlasm
#                                                                       ^ source.hlasm keyword.other.hlasm
#                                                                        ^^^^^^^^ source.hlasm invalid.ignored
>        Length of block does not exceed 256 (&LBL DSECT), &&C          X12345678
#^^^^^^^^^^^^^^^ source.hlasm invalid.illegal
#               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm string.hlasm
#                                             ^^^^ source.hlasm variable.other.readwrite.hlasm
#                                                 ^^^^^^^^^ source.hlasm string.hlasm
#                                                          ^^ source.hlasm string.hlasm
#                                                            ^^^^^^^^^^^ source.hlasm string.hlasm
#                                                                       ^ source.hlasm keyword.other.hlasm
#                                                                        ^^^^^^^^ source.hlasm invalid.ignored
>               LEANUP option can be optimized to YES',OP2OP2OP2OP2OP2O,-12345678
#^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm string.hlasm
#                                                     ^ source.hlasm
#                                                      ^^^^^^^^^^^^^^^^ source.hlasm punctuation.definition.tag.hlasm
#                                                                      ^ source.hlasm
#                                                                       ^ source.hlasm keyword.other.hlasm
#                                                                        ^^^^^^^^ source.hlasm invalid.ignored
>               REMARK REMARK
#^^^^^^^^^^^^^^^ source.hlasm
#               ^^^^^^ source.hlasm punctuation.definition.tag.hlasm
#                     ^ source.hlasm
#                      ^^^^^^ source.hlasm comment.hlasm